composer.json

```diff
@@ -3,7 +3,11 @@
     "description": "Fork of symfony 1.4 with dic, form enhancements, latest swiftmailer and better performance",
     "license": "MIT",
     "require": {
-        "swiftmailer/swiftmailer": "~5.2"
+        "php": ">=5.5.9",
```

Comment:
Upgrading the minimal version of PHP is not something that can be accepted for this repository, as it is focused on long term maintenance.